packages/collector/src/types/collector.d.ts

```diff
@@ -1,4 +1,21 @@
 import { GenericLogger } from '@instana/core/src/core';
+import { IgnoreEndpoints, Disable } from '@instana/core/src/config/types';
```

Comment:
Moving AgentConfig into a ts declaration file requires to move its dependencies to other types too. But its so much better to use the types files anyway.